index.html

```diff
@@ -1,199 +1,172 @@
+                            <section class="butjs"><a class="jsbut" >JavaScript</a></section>
+                        </section></section>
+                    <button class="see">See project</button>
+                </section>
+                </section>
+                <section class="about-page">
+                <section class="about">
+                    <h2 class="myself">About Myself</h2>
+                    <section class="about">
+                        <p class="self">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p></section>
+                    <section class="let">LET’S CONNECT</section>
+                    <section class="about">
+                        <ul class="can">
+                            <li class="can"><a href=""><img class="twitter" src="github.png" alt="twitter"></a></li>
+                            <li class="can"><a href=""><img class="twitter" src="in.png" alt="twitter"></a></li>
+                            <li class="can" ><a href=""><img class="twitter" src="angellist.png" alt="twitter"></a></li>
+                            <li class="can"><a href=""><img class="twitter" src="twitter.png" alt="twitter"></a></li>
+                            <li class="can" ><a href=""><img class="twitter" src="medium.png" alt="twitter"></a></li>
+                        </ul>
+                    </section>
+                    <section class="about">
+                        <button class="get">Get my resume</button>
+                    </section>
+                    <section class="about">
+                        <ul class="down">
+                            <li class="down"><section class="ic"><a class="lang">Languages</a><img class=" vec" src="vect.png"></section>
+                            <li class="down"><section class="ic"><section class="jsfile"><a class="js">JavaScript</a></section></section></li>
+                            <li class="down"><section class="ic"><section class="jsfile"></a><a class="html5">HTML</a></section></section></li>
+                            <li class="down" ><section class="ic"><section class="jsfile"></a><a class="css3">CSS</a></section></section></li></li>
+                            <li class="down"><section class="ic"><a class="lang">Frameworks</a><img class=" vec" src="vect1.png"></section></li>
+                            <li class="down"><section class="ic"><img class="line" src="line.png"></section></li>
+                            <li class="down"><section class="ic"><a class="lang">Skills</a><img class=" vec" src="vect1.png"></section></li>
+                            <li class="down"><section class="ic"><img class="line" src="line.png"></section></li>
+                        </ul>
+                    </section>

```

Comment:
- [ ]  I have noticed that the particular section in question does not appear to be fully displayed on my browser. I kindly suggest adjusting the height of the section to ensure its complete visibility and optimal presentation on mobile view
- [ ] Microveser Design

![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/68823331/839a0db4-c057-446b-936e-e18c7563608f)


- [ ] Your Design

![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/68823331/618d009c-dd96-444b-893f-f989dc82729e)